include/directx/d3d12.h

```diff
@@ -5555,8 +5570,10 @@ EXTERN_C const IID IID_ID3D12DescriptorHeap;
 
 #if !defined(_WIN32)
 
-#define ID3D12DescriptorHeap_GetDesc(This)	\
-    ( (This)->lpVtbl -> GetDesc(This) ) 
+#define ID3D12DescriptorHeap_GetDesc(This,RetVal)	\
+do { \
+   *(RetVal) = ( (This)->lpVtbl -> GetDesc(This) ) \

```

Comment:
as it's non-win32, maybe we can use static inline, as linux always using gcc or gcc-compat compilers.
I think support for inline is not a issue for C language

Reply:
FYI, if this is for mingw there are some ambiguity about using 'inline' functions, see https://sourceforge.net/p/mingw-w64/mailman/message/37666105/

Reply:
This is not for mingw, it's for Linux only.

Reply:
Updated with inline function, so they are very consistence now